We check-in a version of the source tarball internally, so these patches must be
applied during tarball creation. Since the source-build repo only applies
patches during build, these need to live here. 

Generated using the following command (from https://stackoverflow.com/a/36648863):
git submodule --quiet foreach --recursive 'export NAME="${PWD##*/}"; git --no-pager diff --src-prefix="a/submodules/${NAME}/" --dst-prefix="b/submodules/${NAME}/"'

---

diff --git a/submodules/Common/NuGet.config b/Common/NuGet.config
index 09c3488420..c7b9f38d2a 100644
--- a/submodules/Common/NuGet.config
+++ b/submodules/Common/NuGet.config
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <clear/>
     <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
 </configuration>
diff --git a/submodules/FileSystem/NuGet.config b/submodules/FileSystem/NuGet.config
index 1707938..f780022 100644
--- a/submodules/FileSystem/NuGet.config
+++ b/submodules/FileSystem/NuGet.config
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <clear/>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
 </configuration>
\ No newline at end of file